nolabs/features/drug_discovery/get_results.py

```diff
@@ -1,26 +1,78 @@
 from nolabs.features.drug_discovery.data_models.ligand import LigandId
 from nolabs.features.drug_discovery.data_models.result import JobId
 
-from nolabs.api_models.drug_discovery import GetDockingResultDataRequest, GetDockingResultDataResponse
+from nolabs.api_models.drug_discovery import GetDockingResultDataRequest, GetUmolDockingResultDataResponse, \
+    GetDiffDockDockingResultDataResponse, DiffDockLigandMetaData, GetDiffDockLigandSdfRequest, \
+    GetDiffDockLigandSdfResponse
 from nolabs.domain.experiment import ExperimentId
 from nolabs.features.drug_discovery.data_models.target import TargetId
 from nolabs.features.drug_discovery.services.result_file_management import ResultsFileManagement
 
 
-class GetDockingResultsFeature:
+class GetUmolDockingResultsFeature:
```

Comment:
One feature/one file

Reply:
Agree in this case, will fix in the following commit

Note: Some features are closely related (i.e. GetDiffDockResultData and GetDiffDockLigandSdf). I'll leave some of them in the same file (like the ones for management of target data etc.), but in the future development people can separate them into other files by convenience.